animeal/src/Flows/Auth/Modules/Verification/VerificationViewController.swift

```diff
@@ -78,6 +78,7 @@ final class VerificationViewController: BaseViewController {
 
         view.addSubview(resendView)
         resendView.leadingAnchor >= headerView.leadingAnchor
+        resendView.topAnchor >= codeInputView.bottomAnchor + 32.0
         resendView.trailingAnchor <= headerView.trailingAnchor
         resendView.centerXAnchor ~= view.safeAreaLayoutGuide.centerXAnchor
         resendView.topAnchor ~= view.safeAreaLayoutGuide.bottomAnchor - 32.0
```

Comment:
looks like the same value is used at 81 and 84 lines, so maybe it make sense to create meaningful const for such purpose

Reply:
Will fix later with some refactoring